specification/network/resource-manager/Microsoft.Network/stable/2024-10-01/virtualNetworkGateway.json

```diff
@@ -796,6 +796,56 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworkGateways/{virtualNetworkGatewayName}/listRadiusSecrets": {
+      "post": {
+        "tags": [
+          "VirtualNetworkGateways"
+        ],
+        "operationId": "VirtualNetworkGateways_ListRadiusSecrets",
+        "description": "List all Radius servers with respective radius secrets from virtual network gateway VpnClientConfiguration.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "virtualNetworkGatewayName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the virtual network gateway."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
```

Comment:
Confusing names:

Why is the API called 'list radius secrets', but the response type is 'RadiusAuthServerListResult'?

Are you listing secrets, auth servers, or what??